wagtail/core/models.py

```diff
@@ -1756,6 +1756,11 @@ def copy(self, recursive=False, to=None, update_attrs=None, copy_revisions=True,
         :param log_action flag for logging the action. Pass None to skip logging.
             Can be passed an action string. Defaults to 'wagtail.copy'
         """
+
+        # If the model doesn't exist in the DB, the operation will not work
```

Comment:
The operation does work, it's just weird that it does. Maybe get rid of this comment as the code itself reads quite well